spec/infinispan_local_spec.js

```diff
@@ -5,370 +5,274 @@ var readFile = Promise.denodeify(require('fs').readFile);
 var f = require('../lib/functional');
 var t = require('./utils/testing'); // Testing dependency
 
-describe('Infinispan local client', function() {
-  var client = t.client(t.local);
+function describeInfinispanBehaviourInLocalModeWithDifferentHotrodProtocol(options) {
+  describe('with hotrod protocol: ' + options.version, function () {
+    var client = t.client(t.local, null, options.version);
 
-  beforeEach(function(done) { client
```

Comment:
Yeah I agree @galderz . I was thinking about that as well.
Well the simplest solution would be to set some environment variable with protocol version, and then use it in the spec with process.env.VAR_NAME  reference. 
I will look closer into the arguments provided by jasmine-node to find out if it has anything which I can apply.